### ISSUE TITLE
KubeCon Europe 2023

### ISSUE THREAD

**Issue by roberthstrand** [amestofortytwo/blog#40]
There are two potential blog posts we can do for KubeCon EU 2023. One before, where we talk about our plans for the event, and one after that summarize what we saw and experienced, as well as key takeaways.

**Comment by alegri90** [comment]
Completely agree - One each, or a bigger one together? 😉

**Comment by roberthstrand** [comment]
I'm thinking one each. If we wait until end of February with the first one, we would then know more details on what's going to happen. The second one should be easy to deal with after we get back.

**Comment by alegri90** [comment]
Sounds like a plan! 👍

Anecdote: 
After the 2019-event I did an internal (company-owned / private) podcast-type of recording, where I talked about the experience and the tech, and how we could and should use it. The target audience at that time was about 50 techs who were about to start their journey to understand Docker and Kubernetes. 

Was fun to do 😊

**Comment by roberthstrand** [comment]
We could combine the blog post with an event. Instead of doing it internally, which we probably would have done anyway, we could host it as a meetup.

**Comment by alegri90** [comment]
Yeah, I was thinking something along the lines of a meetup for a watch-and-talk by screening and discussing some of the really good presentations. 

Like they are doing today at this IxDA Stavanger event: https://www.meetup.com/ixda-stavanger/events/290740371/
